packages/react/src/components/Table/Table.story.jsx

```diff
@@ -2304,7 +2335,7 @@ export const WithStickyHeaderExperimentalAndCellTooltipCalculation = () => {
         id="table"
         columns={tableColumns.map((i) => ({
           ...i,
-          renderDataFunction,
+          renderDataFunction: i.renderDataFunction || renderDataFunction,
```

Comment:
don't overwrite the renderDataFunction in this story if it already exists (like for the new object column)